AGPs/AGP-CRDAO.md

```diff
@@ -0,0 +1,60 @@
+---
+AGP: N/A
+Title:  Community Rewards DAO
+Author: Aaron Foster (`@pythonpete32`) & Burrrata (`@burrrata`)
+Status: Stage III
+Track: Finance
+Created: 2019-10-01
+---
+
+# AGP X: Community Rewards DAO
+
+## Address of the transfer recipient
+
+TBD, but will be the Vault of the CRDAO
+
+## Amount of the transfer
+
+12,000 ANT
+- 1000 ANT for every week between this upcoming AGP (fourth Thursday of October) and the next AGP (fourth Thursday of January)
```

Comment:
There's an AGP that could be approved this ANV which would make the next ANV the second week of December. Please clarify how this would impact the AGP

Reply:
Removed this part so that there is no confusion.